addons/project/static/src/components/subtask_kanban_list/subtask_kanban_list.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<templates>
+
+    <t t-name="project.SubtaskKanbanList" owl="1">
+       <Record resModel="'project.task'" resId="props.record.resId" fields="fieldsInfo" activeFields="fieldsInfo" t-slot-scope="data">
+            <div class="subtask_list">
+                <t t-foreach="data.record.data.child_ids.records" t-as="subTask" t-key="subTask.resId">
+                    <div class="subtask_list_row">
+                        <div class="subtask_state_widget_col d-flex justify-self-center align-self-center">
+                            <Field name="'kanban_state'" 
+                                   record="subTask" 
+                                   fieldInfo="subTask.fields.kanban_state" 
+                                   type="'state_selection'"/>
+                        </div>
```

Comment:
```suggestion
                        <Field name="'kanban_state'" 
                               class="'subtask_state_widget_col d-flex justify-self-center align-self-center'"
                               record="subTask" 
                               fieldInfo="subTask.fields.kanban_state" 
                               type="'state_selection'"/>
```
Maybe we could pass the classes like this to avoid adding a parent div. 🙂